tests/test_schema.py

```diff
@@ -350,6 +350,50 @@ def test_load_many():
     assert type(result[0]) == User
     assert result[0].name == 'Mick'
 
+def test_load_many_input_types():
```

Comment:
This is largely redundant with `test_non_iterables_with_many`. Maybe remove this in favor of that one, and separate out the empty collection cases (`s.load([])` and `s.load(set())`) into a separate test function.